### ISSUE TITLE
Add a cropping tool to the user profile image uploader

### ISSUE THREAD

**Issue by BjarniRunar** [piratar/wasa2il#121]
The UI assumes in many places that images are square, and forces them to be square if they are not.

This makes people look squashed and silly in elections, and maybe elsewhere.

Some clever chap should totally add a cropper that lets the user upload a photo and then select their own favorite square to use on the site.

**Comment by PeterTheOne** [comment]
It might be enough to not stretch but instead crop images in css.

**Comment by nifgraup** [comment]
Here is a script to recrop thumbnails as actual squares.

```
#!/usr/bin/env bash

# npm install smartcrop-cli
SMARTCROP=$PWD/node_modules/.bin/smartcrop

pushd wasa2il/uploads
ls userimg_*.jpg | grep -o "userimg_[0-9a-f]*" | uniq | xargs -n 1 -I{} \
$SMARTCROP --width 50 --height 50 {}.jpg {}_50x50.jpg
popd
```

**Comment by viktorsmari** [comment]
@nifgraup Will this result in the right half of the candidates face being removed, if the uploaded picture is wide? 

Another quick solution is just disabling 'height:50px' and 'width:50px' css for .vote-image, but then the result is like this:
![2016-08-02_128x210](https://cloud.githubusercontent.com/assets/1689020/17317775/4c4e5cce-5881-11e6-8841-380939802b83.png)

Maybe this is a better alternative then having skewed images?

I guess the original problem is not asking users to crop the image and selecting a rectangle at the upload process.

**Comment by nifgraup** [comment]
@viktorsmari no, that should not happen https://github.com/jwagner/smartcrop.js#algorithm-overview. I tested and it works well.

I'm just suggesting as a quick fix, I understand this doesn't fix the original issue.

**Comment by viktorsmari** [comment]
@nifgraup Looks good!

**Comment by nifgraup** [comment]
I opened  a pull request for some progress, note that `wasa2il` generates thumbnails on the fly so to test it's enough to delete `wasa2il/uploads/userimg_*50x50*` and refresh.

**Comment by 7oi** [comment]
Merged in #130
